packages/clerk-js/src/ui/components/UserButton/UserButtonTrigger.tsx

```diff
@@ -20,6 +20,9 @@ export const UserButtonTrigger = withAvatarShimmer(
         variant='roundWrapper'
         sx={[theme => ({ borderRadius: theme.radii.$circle }), sx]}
         ref={ref}
+        aria-label='user-button-trigger'
```

Comment:
This will also need a better `aria-label` name.

Also an `aria-controls`

Reply:
Same as above